include/wx/uilocale.h

```diff
@@ -40,6 +41,16 @@ class WXDLLIMPEXP_BASE wxLocaleIdent
     // and region separated by dashes.
     static wxLocaleIdent FromTag(const wxString& tag);
 
+    // Add likely subtags to a given locale identifier.
+    static wxLocaleIdent AddLikelySubtags(const wxLocaleIdent& localeIdent);
+
+    // Remove likely subtags from a given locale identifier, favor region.
+    static wxLocaleIdent RemoveLikelySubtags(const wxLocaleIdent& localeIdent, bool favorRegion = true);
+
+    // Find the best match between desired and supported languages/locales.
```

Comment:
This API is error-prone because it's simple to exchange the parameters by mistake. Maybe it's not really important if it's mostly for internal use anyhow and only used in 1 or 2 places, but if we could make it less symmetrical, it would be nice...